src/javascripts/components/cards/equipmentCards.js

```diff
@@ -3,19 +3,25 @@ import 'firebase/auth';
 import equipmentData from '../../helpers/data/equipmentData';
 
 const authedEquipmentCardView = (equipmentObject) => {
-  const domString = `<div>
+  const domString = `<div class="card card-body" id="${equipmentObject.equipmentId}">
+      <div>
         <img src="${equipmentObject.imageUrl}" class="card-img-top" alt="${equipmentObject.name}">
         <div>
           <h3 class="card-text card-header">${equipmentObject.name}</h3>
         </div>
-        <button type="button" id="${equipmentObject.equipmentId}" class="btn btn-info update-equipment">Edit</button>
+        <button type="button" id="${equipmentObject.equipmentId}" class="btn btn-info update-equipment card-btns"><i class="fas fa-pen"></i></button>
+        <button type="button" id="${equipmentObject.equipmentId}" class="btn btn-info delete-equipment card-btns"><i class="fas fa-trash-alt"></i></button>
       </div>
     </div>`;
+  $('body').on('click', 'button.delete-equipment', (e) => {
+    $(`.card#${e.currentTarget.id}`).remove();
```

Comment:
in staying consistent with your other on click events, maybe consider also storing e.currentTarget.id into firebaseKey variable

Reply:
done!